packages/fxa-auth-server/scripts/template-version-bump.js

```diff
@@ -10,16 +10,18 @@
 // a deleted template is reinstated by some later commit or only one
 // format of a template is deleted.
 
+// TODO: account for sms/templates
```

Comment:
I need to remove this comment since we're going to remove SMS stuff later, made this originally as a note to myself when fixing up SMS rendering/tests